js/start.md

```diff
@@ -275,454 +229,570 @@ To install React-specific Amplify UI components, run the following command:
     "types" : ["node"]
 }
 ```
 
-In your [app module](https://angular.io/guide/bootstrapping) `src/app/app.module.ts`, change your code to the following:
+Note: If you are using Angular 6 or above, you may need to add the following to the top of your `src/polyfills.ts` file: 
+```
+(window as any).global = window;
 
-```javascript
-import { BrowserModule } from '@angular/platform-browser';
-import { CommonModule } from '@angular/common';
-import { NgModule } from '@angular/core';
-
-import { AppComponent } from './app.component';
-import { AmplifyAngularModule, AmplifyService } from 'aws-amplify-angular';
-
-@NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    CommonModule,
-    AmplifyAngularModule
-  ],
-  providers: [
-    AmplifyService
-  ],
-  bootstrap: [AppComponent]
-})
-
-export class AppModule { }
-```
-
-This imports the Amplify Module and Service.
-
-Note: If you are using Angular 6 or above, you may need to add the following to the top of your `src/polyfills.ts` file: ```(window as any).global = window;```.
+(window as any).process = {
+  env: { DEBUG: undefined },
+};
+```
 {: .callout .callout--info}
 
-In your `src/app/app.component.ts` file, add the following import statements:
+In your `src/app/app.component.ts` file, add the following imports and modifications to your class to to add data to your database with a mutation by using the `API.service` file which was generated when you ran `amplify add api`:
 
 ```javascript
-import { AmplifyService } from 'aws-amplify-angular';
-import awsconfig from '../aws-exports';
+import { APIService } from './API.service';
+
+export class AppComponent {
+
+  constructor(private apiService: APIService) {}
+
+  createTodo() {
+    this.apiService.CreateTodo({
+        name: 'Angular',
+        description: 'testing'
+    });
+  }
+}
 ```
 
-To add the analytics event recorder to your app, replace your ```AppComponent``` class with the following:
+Add the following to your `src/app/app.component.html` to add a button which calls your `createTodo()` method:
+
+```html
+<button (click)="createTodo()">Add Todo</button>
+```
+
+Next, update the class to list all items in the database by running a `ListTodos` query when the app starts by implementing [OnInit](https://angular.io/api/core/OnInit) and storing the items in an array:
 
 ```javascript
-export class AppComponent {
-  title = 'amplify-angular-app';
-  url = 'https://console.aws.amazon.com/pinpoint/home/?region='
-        + awsconfig.aws_project_region + '#/apps/'
-        + awsconfig.aws_mobile_analytics_app_id + '/analytics/events';
-  eventsSent = 0;
-  analyticsEventSent = false;
-
-  constructor( private amplifyService: AmplifyService ) {}
-
-  handleAnalyticsClick() {
-    this.amplifyService.analytics().record('AWS Amplify Tutorial Event')
-    .then( (evt) => {
-        ++this.eventsSent;
-        this.analyticsEventSent = true;
+
+export class AppComponent implements OnInit {
```

Comment:
It might be worth showing:
`import { Component, OnInit } from '@angular/core';`